### DISCUSSION TITLE
Don't show tooltip when the label value is empty?

### DISCUSSION THREAD

**Discussion by Toshinaki** [mantinedev/mantine#5539] [Feature requests]
I have a use case that renders tags as `Badge`, with tag name shows in the badge, and `Tooltip` showing the tag's description.

However the description is optional, so tags with description set show tooltip properly, 
but tags without description also show the tooltip, but only a black box with no content
something like this
![image](https://github.com/mantinedev/mantine/assets/10347466/bede4a94-2da4-4e44-991d-b1f85c85851c)

It would be great for tooltip not showing up, when the content is empty (`""`, `null` or `undefined`)

**Comment by rtivital** [comment]
`<Tooltip disabled={!yourLabel} label={yourLabel} />`

**Reply by Toshinaki** [reply]
Thanks! It would be great if we could have an example in the doc.
